sdcm/microbenchmarking.py

```diff
@@ -369,7 +375,7 @@ def main(args):
         if args.before_date:
             mbra.exclude_before_date(args.before_date)
     if args.mode == 'check':
-        mbra = MicroBenchmarkingResultsAnalyzer(email_recipients=args.email_recipients.split(","))
+        mbra = MicroBenchmarkingResultsAnalyzer(email_recipients=args.email_recipients.split(","), build_url=args.build_url, job_url=args.job_url)
```

Comment:
MicroBenchmarkingResultsAnalyzer shouldn't be aware of how to get the URLs, so:
`build_url=os.environ.get('BUILD_URL', ""), job_url=...`

Reply:
removed